src/main/java/ch/njol/skript/sections/SecWhile.java

```diff
@@ -126,4 +240,31 @@ public void exit(Event event) {
 		super.exit(event);
 	}
 
+	private Conditional<Event> getConditional() {
+		return conditional;
+	}
+
+	private boolean isDoWhile() {
+		return doWhile;
+	}
```

Comment:
```suggestion
	private Conditional<Event> getConditional() {
		return conditional;
	}

	private boolean isDoWhile() {
		return doWhile;
	}
```
Are these needed? I think you should be able to reference the fields directly.